services/apps/alcs/src/portal/application-submission/application-owner/application-owner.service.ts

```diff
@@ -290,6 +292,17 @@ export class ApplicationOwnerService {
           await this.applicationSubmissionService.update(submissionUuid, {
             applicant: applicantName || '',
           });
+
+          const fileNumber =
+            await this.applicationSubmissionService.getFileNumber(
+              submissionUuid,
+            );
+          if (fileNumber) {
+            await this.applicationService.updateApplicant(
+              fileNumber,
+              applicantName || 'Unknown',
```

Comment:
nitpick: 'Unknown' is perfect candidate for constant

Reply:
Good call, there were actually like 7 of them

Reply:
Made a FALLBACK_APPLICANT_NAME const and used everywhere in the backend